docs/tutorials/delete-a-task_chood.md

```diff
@@ -0,0 +1,42 @@
+# Tutorial: Delete a task
+
+Here you will learn how to delete a task from your to-do list.
+
+This tutorial takes about 10 minutes to complete.
+
+**Caution**: Deleting a task removes the task item from the To-do Service forever.
+
+## Prerequisites
+
+Make sure to complete all of the steps in [System Pre-requisites](../before-you-start-a-tutorial.md).
+
+## Delete a task
+
+Deleting a task uses a request with the (`DELETE`) method and the task's task ID.
+
+### To delete a task
+
+1. Confirm your local service is running. If it's not, use the following command to start it:
+
+    ```shell
+    cd your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+2. Open the Postman app on your desktop.
+3. In the Postman app, create a new request with the following values:
+   * **METHOD**: DELETE
+   * **URL**: `{% raw %}{{base_url}}/tasks{% endraw %}`
```

Comment:
I added the `{% raw %}` and `{% endraw %}` notation around `{{base_url}}` to render it. The `{{` and `}}` are interpreted as document generation command and so it doesn't render the text to the final page. 
The `{% raw %}` and `{% endraw %}` tags tell the document generator to treat what's between them as plain text and not commands.